geowave/src/main/scala/geotrellis/spark/io/kryo/GeowaveKryoRegistrator.scala

```diff
@@ -36,13 +39,46 @@ class GeowaveKryoRegistrator extends KryoRegistrator {
     }
   }
 
-  // Serializer for GridCoverage2D.  Delegate to Java Serialization.
-  private class GridCoverage2DSerializer extends Serializer[GridCoverage2D] {
-    override def write(kryo: Kryo, output: Output, gc: GridCoverage2D): Unit = {
+  /**
+    * SimpleFeatureType serializer.  This makes use of the
+    * encoding/decoding machinery provied by GeoTools.
+    */
+  private class SimpleFeatureTypeSerializer extends Serializer[SimpleFeatureType] {
```

Comment:
I decided to remove the custom serializer and replace it with the javakaffe one.